src/derivatives/atlas.md

```diff
@@ -548,7 +501,7 @@ A guide for using macros can be found at
 })
 }}
 
-where the `atlas-RamonCajal1908` and `atlas-Economo1916` hypothetically define
+where the `seg-RamonCajal1908` and `seg-Economo1916` hypothetically define
```

Comment:
This conflates seg with atlas. Seg is not anymore a segmentation, but a family of annotated knowledge of the brain.